PINRemoteImage.podspec

```diff
@@ -19,12 +19,22 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '6.0'
   s.requires_arc = true
+  
+  # Exclude optional FLAnimatedImage module
+  s.default_subspec = 'Core'
```

Comment:
Will this make the default behaviour of the pod be to exclude FLAnimatedImage instead of including it - will this have the effect of `pod update` breaking clients?